docs/_static/nav-config.js

```diff
@@ -0,0 +1,41 @@
+const navConfig = {
+
+    "navTextLinks": [
+        {
+            "title": "API Docs",
+            "href": "../api-docs",
+            "pathMatch": "somewhere",
+        },
+        {
+            "title": "Blog",
+            "href": "../blog/",
+            "pathMatch": "somewhere",
+        },
+        {
+            "title": "User Guide",
+            "href": "../user-guide",
+            "pathMatch": "somewhere",
+        },
+    ],
+    "navProductName": "TKET",
+    "navIconLinks": [
+        {
+            "title": "TKET Github",
+            "href": "https://github.com/CQCL/tket",
+            "pathMatch": "somewhere",
+            "iconImageURL": "_static/assets/github.svg",
+        },
+        {
+            "title": "TKET Slack Channel",
+            "href": "https://tketusers.slack.com/",
```

Comment:
Is this a valid invite link?

Reply:
I checked this and it seems to go to the correct page for people not in the workspace.